gptqmodel/utils/model.py

```diff
@@ -633,3 +634,18 @@ def check_to_quantized(config):
         if config.bits > 8 or "fp" in config.data_type or "float" in config.data_type:
             return False
         return True
+
+def copy_remote_file(save_dir, file_extension=".py", model_id_or_path=""):
```

Comment:
rename to more accurate`copy_py_files` since we actually don't have a way to detect what files are for `trust_remote_code`.